src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("fruit,quantity")
+                .append(System.lineSeparator());
+        for (Fruit name: Storage.keyset()) {
+            Fruit key = name;

```

Comment:
The variable 'key' is redundant, as you can use the variable 'name' directly.

Reply:
ok, done